library/src/scripts/components/mebox/pieces/VanillaHeaderNav.tsx

```diff
@@ -0,0 +1,40 @@
+/*
+ * @author Stéphane LaFlèche <stephane.l@vanillaforums.com>
+ * @copyright 2009-2018 Vanilla Forums Inc.
+ * @license GPL-2.0-only
+ */
+
+import * as React from "react";
+import classNames from "classnames";
+import VanillaHeaderNavItem, { IVanillaHeaderNavItemProps } from "./VanillaHeaderNavItem";
+
+export interface IVanillaHeaderNavProps {
+    className?: string;
+    children: IVanillaHeaderNavItemProps[];
+    linkClassName?: string;
+    linkContentClassName?: string;
+}
+
+/**
+ * Implements Navigation component for header
+ */
+export default class VanillaHeaderNav extends React.Component<IVanillaHeaderNavProps> {
+    public render() {
+        const content = this.props.children.map((item, key) => {
+            return (
+                <React.Fragment key={`headerNavItem-${key}`}>
```

Comment:
This fragment can be removed and the key put directly on the `VanillaHeaderNavItem`. Key's also don't need readable names or to be unique across the whole application. They're purely for react's internal use and only need to be unique within their own iteration.